codeforamerica/vita-min
# Fix mdEL101 spec failure in the evening again

[It's...back again:
](https://app.circleci.com/pipelines/github/codeforamerica/vita-min/27794/workflows/248ada5c-21b2-4bf6-a5e2-4fded9b403e5/jobs/76659/tests
)
[See that it wasn't an issue (this PR didn't have anything to do with MD EL101) but it starts being an issue ~9PM PT
](https://app.circleci.com/pipelines/github/codeforamerica/vita-min?branch=FYST-1469-update-misc-nc-copy-for-translations)
Taking a page out of NJ playbook to set any timestamp we manually set to a noon at UTC timestamp so that it will always be converted to the same day. [See q & a here](https://github.com/codeforamerica/vita-min/pull/5264#discussion_r1895942471)

## Link to pivotal/JIRA issue
- N/A
## Is PM acceptance required? (delete one)
- No - merge after code review approval

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Setting the signed_at timestamp to noon time will ensure it will always return the same Date because the UTC/other timezone difference will be within the boundaries of changing days (~7 hours difference between UTC / Arizona time zone)
## How to test?
- Run tests at all hours & make sure it does not fail again
- Risk Assessment
  - Not any really -- just trying to get tests to consistently pass regardless of when its run
